src/main/java/bio/overture/ego/event/token/CleanupTokenListener.java

```diff
@@ -61,7 +61,7 @@ private void cleanupTokensForUser(@NonNull User user) {
     tokens.forEach(t -> verifyToken(t, scopes));
   }
 
-  private void verifyToken(@NonNull TokenResponse token, @NonNull Set<String> scopes) {
+  private void verifyToken(@NonNull ApiKeyResponse token, @NonNull Set<String> scopes) {
```

Comment:
lets rename this method as well. Actually this class can be renamed to `CleanupApiKeyListener`

Reply:
also lets rename the `token` variable :)

Reply:
derp, good point!